Library/Formula/yelp-tools.rb

```diff
@@ -0,0 +1,20 @@
+class YelpTools < Formula
+  homepage "https://github.com/GNOME/yelp-tools"
+  desc "Tools that help create and edit Mallard or DocBook documentation."
+  url "https://download.gnome.org/sources/yelp-tools/3.16/yelp-tools-3.16.1.tar.xz"
+  sha256 "b4f66c145af1c6448dc51037d305d6844da13dc31d07729b8e29005ee4fef89c"
+
+  depends_on "itstool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "yelp-xsl" => :build
+
+  def install
+    system "./configure", "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/yelp-check", "--help"
```

Comment:
doubt that's possible: these executables need specifically formatted files to work on, which can be found in gnome projects.

I only added these formulas because I needed them to autogen.sh glade3, for which I intend to submit a formula when the gtk to quartz transition is finished.